### PR TITLE
Move scribbles to generic segmentation app

### LINKED ISSUE #312: Scribbles - Move scribbles to the generic segmentation app
As discussed in weekly status meeting, this is to track moving all relevant code from spleen scribbles app to the generic segmentation example at: 
https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/generic_segmentation

This is to show a consolidated example that contains scribbles along with other monailabel interactive segmentation features.

### PR DESCRIPTION
**Pull request by masadcv**
Fixes #312 

Moving scribbles methods to generic segmentation app, a followup pull request may be required to move spleen_segmentation_scribbles app to sample apps repository.

### CODE REVIEW

File: sample-apps/generic_segmentation/main.py [hunk 1]
```diff
@@ -59,6 +66,11 @@ def __init__(self, app_dir, studies):
     def init_infers(self):
         infers = {
             "segmentation": MyInfer([self.pretrained_model, self.final_model], self.network),
+            "Coldstart->ISeg+GraphCut": GenericISegGraphcutColdstart(),
+            "ISeg+GraphCut": GenericISegGraphCut(),
+            "ISeg+SimpleCRF": GenericISegSimpleCRF(),
+            # "ISeg+CRF": GenericISegCRF(), # disabled for now as MONAI CRF GPU has a bug
```
Comment (SachidanandAlle) [inline sample-apps/generic_segmentation/main.py:72]: lets remove these lines.. lets not confuse external users when we add them to main branch

**Comment by SachidanandAlle** [comment]
initial draft looks good to me.. lets also plan to remove the segmentation_spleen_scribbles app (we can move that app to say https://github.com/diazandr3s/MONAILabel-Apps)

File: sample-apps/generic_segmentation/main.py (hunk 1 above)
Comment (masadcv) [reply sample-apps/generic_segmentation/main.py:72]: Sure, will remove it

**Comment by masadcv** [comment]
> initial draft looks good to me.. lets also plan to remove the segmentation_spleen_scribbles app (we can move that app to say https://github.com/diazandr3s/MONAILabel-Apps)

Many thanks for reviewing. Yes, I agree - we can move the scribbles app to sample apps repository. I will look into it after finishing with this pull request.

**Comment by masadcv** [comment]
This is now ready for review/merge.

**Review by SachidanandAlle — APPROVED** [review]

**Pull request merged**